index.html

```diff
@@ -1,40 +1,138 @@
-      Write to us at <a href='mailto:moneyskills@scripbox.com'>moneyskills@scripbox.com</a>
+  </div>
+  <div class="container-fluid">
+    <div class="container">
+      <div class="caption-2">
+        <p class="text-center mt-5 mb-2">Scripbox Money Skills Program is designed to effectively engage your teams and prepare them to make their money work
+          harder
+        </p>
+      </div>
+      <div class="sub-heading">
+        <p class="text-center mt-5 mb-3">Our 45 minutes introductory session will talk about</p>
+      </div>
+    </div>
+    <div class="container">
+      <div class="row mb-5">
+        <div class="col-md-6 text-center">
+          <img src="images/inflation.png" alt="inflation">
+        </div>
+        <div class="col-md-6">
+          <p class="caption-2">Inflation</p>
+          <p class="sub-heading">How the value of money decreases day by day.</p>
+        </div>
+      </div>
+    </div>
+    <div class="container image-content">
+      <div class="row mb-5">
+        <div class="col-md-6 push-md-6 text-center">
+          <img src="images/compounding.png" alt="compounding">
+        </div>
+        <div class="col-md-6 pull-md-6">
+          <p class="caption-2">Compounding</p>
+          <p class="sub-heading">The way to grow your money.</p>
+        </div>
+      </div>
+    </div>
+    <div class="container image-content">
+      <div class="row mb-5">
+        <div class="col-md-6 text-center">
+          <img src="images/invest.png" alt="invest">
+        </div>
+        <div class="col-md-6">
+          <p class="caption-2">Why Invest?</p>
+          <p class="sub-heading">Know where to invest and how to save your money.</p>
+        </div>
+      </div>
+    </div>
+    <div class="container image-content">
+      <div class="row mb-5">
+        <div class="col-md-6 push-md-6 text-center">
+          <img src="images/4-kinds-money.png" alt="4 kinds of money">
+        </div>
+        <div class="col-md-6 pull-md-6">
+          <p class="caption-2">Four kinds of money</p>
+          <p class="sub-heading">Understanding the four kinds of money-and how to manage each.</p>
+        </div>
+      </div>
+    </div>
+    <div class="container image-content">
+      <div class="row">
+        <div class="col-md-6 text-center">
+          <img src="images/get-started.png" alt="get started">
+        </div>
+        <div class="col-md-6">
+          <p class="caption-2">Get started</p>
+          <p class="sub-heading">Know where to invest and how to save your money.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container-fluid no-padding" id="form-section">
+    <div class="section-3">
+      <div class="details-form col-md-6 px-4 py-5" id="myform">
+        <p class="caption-2 mt-2 mb-5 text-center">Register to Money Skills Program</p>
+        <iframe name=“leadsquared_landing_page_frame” src=“http://scripbox1.viewpage.co/Landing-Page-02?ignoremxtracking=mxtrue”
```

Comment:
Iframe embed form isn't working.. We need to speak with leadsquared to resolve it.